Packs/Gmail/Integrations/Gmail/Gmail_test.py

```diff
@@ -698,22 +703,56 @@ def test_fetch_incidents(mocker, return_value_get_last_run, expected_result):
     assert incidents == expected_result
 
 
-def test_get_occurred_date():
+email_no_internalDate = input_data.email_without_date
```

Comment:
```suggestion
EMAIL_NO_INTERNALDATE = input_data.email_without_date
```
And lets change all the global vars as follows